features/admin/product/managing_products/product_validation.feature

```diff
@@ -29,18 +29,19 @@ Feature: Products validation
         And I try to save my changes
         Then I should be notified that the locale is not available
 
-    @todo @ui @no-api
+    @ui @mink:chromedriver @no-api
     Scenario: Adding a new simple product with duplicated code among products
         Given the store has a product "7 Wonders" with code "AWESOME_GAME"
         When I want to create a new simple product
         And I specify its code as "AWESOME_GAME"
         And I name it "Dice Brewing" in "English (United States)"
+        And I generate its slug in "English (United States)"
```

Comment:
In some scenarios, like this one, in order not to have to add the `javascript`/`mink:chromedriver` tag, it is better to set a slug instead of generating

Reply:
Changes applied